src/Forms/iOS/packages.config

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Esri.ArcGISRuntime.Hydrography" version="100.3.0" targetFramework="xamarinios10" />
-  <package id="Esri.ArcGISRuntime.Xamarin.Forms" version="100.3.0" targetFramework="xamarinios10" />
-  <package id="Esri.ArcGISRuntime.Xamarin.iOS" version="100.3.0" targetFramework="xamarinios10" />
```

Comment:
We'll need to be careful to make sure the references to 100.3 (non-daily) are kept.